unitary/alpha/quantum_world_test.py

```diff
@@ -673,6 +677,10 @@ def test_get_histogram_and_get_probabilities_one_binary_qobject(
     assert len(histogram[0]) == 2
     assert histogram[0][0] > 10
     assert histogram[0][1] > 10
+    histogram = world.get_correlated_histogram()
+    assert len(histogram) == 2
+    assert histogram[(0,)] > 10
+    assert histogram[(1,)] > 10
```

Comment:
Should we have a test where the correlated histogram actually captures some entanglement and is different than the normal histogram?

Reply:
Thanks Doug! A new test is added.